schema/settings.schema.json

```
@@ -341,6 +341,14 @@
             "pointer": true
           }
         },
+        "enableGoImportsSearchQueryTransform": {
+          "description": "Enables `go.imports:` search expansion (formally known as go imports search extension).",
```

```suggestion
          "description": "Lets you easily search for all files using a Go package. Adds a new operator `go.imports`: for all import statements of the package passed to the operator.",
```
A hint about what the search expansion actually does (taken from the [original repo](https://github.com/sourcegraph/go-imports-search#go-imports-search)).